spec.bs

```diff
@@ -3050,45 +3056,65 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   : <dfn>is chaff</dfn>
   :: A [=boolean=] field indicating whether this response should be ignored.
   : <dfn>ad render url</dfn>
-  :: The [=URL=] of the ad for the winning bid.
+  :: [=URL=]. The [=leading bid info/leading bid=]'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=] from the
+     component auction executed on the trusted auction server.
   : <dfn>ad components</dfn>
-  :: A [=list=] of [=URLs=] containing the ad components for the winning bid.
+  :: A [=list=] of [=URLs=]. A list of [=leading bid info/leading bid=]'s
+     [=generated bid/ad component descriptors=]'s [=ad descriptor/urls=] from the
+     component auction executed on the trusted auction server.
   : <dfn>interest group name</dfn>
-  :: The winning interest group's name as a [=string=].
+  :: The [=leading bid info/leading bid=]'s [=generated bid/interest group=]'s
+     [=interest group/name=] as a [=string=] from the
+     component auction executed on the trusted auction server.
   : <dfn>interest group owner</dfn>
-  :: The winning interest group's owner [=origin=].
+  :: An [=origin=]. The [=leading bid info/leading bid=]'s
+     [=generated bid/interest group=]'s [=interest group/owner=] from the
+     component auction executed on the trusted auction server.
   : <dfn>bidding groups</dfn>
-  :: A [=list=] of interest group [=interest group/owner=] and interest group [=interest group/name=] pairs that bid
+  :: A [=list=] of [=generated bid/interest group=] [=interest group/owner=] and
+     [=generated bid/interest group=] [=interest group/name=] pairs that bid

```

Comment:
it's fine to not link to interest group since owner/name already does.
```suggestion
  :: A [=list=] of interest group [=interest group/owner=] and
     interest group [=interest group/name=] pairs that bid
```